### ISSUE TITLE
DataTableMenu is not keyboard/tab accessible

### ISSUE THREAD

**Issue by vpicone** [carbon-design-system/carbon#2481]
- [ ] the focus menu itself should be focused when clicked
- [ ] when activated via keyboard, the first menuItem should receive focus
![Apr-22-2019 09-44-22](https://user-images.githubusercontent.com/40970507/56506156-3de0b080-64e3-11e9-9140-26a671557749.gif)

Focus management is currently a big issue for the OverflowMenu, especially when it is used in a compound component. We need to start work on an accessible replacement or not use that component for the DataTableMenu.
